Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp

```diff
@@ -1800,7 +1800,7 @@ Ztring MediaInfo_Config_MediaInfo::File_Duplicate_Get (size_t AlreadyRead_Pos)
     CriticalSectionLocker CSL(CS);
     if (AlreadyRead_Pos>=File__Duplicate_List.size())
         return Ztring(); //Nothing or not more than the last time
-    Ztring Temp=File__Duplicate_List[AlreadyRead_Pos];
+    const Ztring Temp=File__Duplicate_List[AlreadyRead_Pos];

```

Comment:
why not return File__Duplicate_List[AlreadyRead_Pos]; here?
(I missed something?)